travis-ci/cleanup-integration-tests.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/sh
+
+export AWS_ACCESS_KEY_ID="$INTEGRATION_AWS_ACCESS_KEY_ID"
+export AWS_SECRET_ACCESS_KEY="$INTEGRATION_AWS_SECRET_ACCESS_KEY"
+export AWS_DEFAULT_REGION="$INTEGRATION_AWS_DEFAULT_REGION"
+
+if [ -z "$DEPLOYMENT" ]; then
+  DEPLOYMENT=$(node ./travis-ci/select-stack.js)
+  if [ "$DEPLOYMENT" = "none" ]; then
+    echo "Unable to determine integration stack" >&2
+    exit 1
+  fi
+fi
+export DEPLOYMENT
+
+# Wait for the stack to be available
+KEY="travis-ci-integration-tests/${DEPLOYMENT}.lock"
+
+# Delete the stack if it's a nightly build
+if [ "$DEPLOYMENT" = "cumulus-nightly" ]; then
+  cd example
```

Comment:
This should be `cd example || exit 1`, in case the cd command fails.

https://github.com/koalaman/shellcheck/wiki/SC2164